src/Components/Auth/Login.tsx

```diff
@@ -1,5 +1,5 @@
 import { useEffect, useState } from "react";
-import { useDispatch } from "react-redux";
+import useQuery from "../../Utils/request/useQuery";
 import { postForgotPassword, postLogin } from "../../Redux/actions";
```

Comment:
When migrating, you'll have to replace these actions with the API routes instead.
You'll need to add the response types to those API routes too if not present.

Have a look at the example PR and documentation attached with the EPIC issue.